Ical.Net/Evaluation/RecurrencePatternEvaluator.cs

```diff
@@ -900,6 +900,15 @@ private Period CreatePeriod(DateTime dt, IDateTime referenceDate)
 
         public override HashSet<Period> Evaluate(IDateTime referenceDate, DateTime periodStart, DateTime periodEnd, bool includeReferenceDateInResults)
```

Comment:
While we update the code, let's also add missing or update existing XmlDoc (for us and for others).

Reply:
Basically a great idea, but also quite challenging. In some cases we can work on details of the code without having a comprehensive understanding of the code as a whole. But the latter is needed for writing a meaningful doc comment. In the given case I wouldn't feel comfortable with writing a doc comment, as I don't have sufficient understanding of the code as a whole. And a bad comment could be misleading and therefore counterproductive.

Reply:
Let's say, add docs as far as we believe to understand :)
No need to be complete in all aspects, but should contain what we found out.